python/psqlpy/_internal/__init__.pyi

```diff
@@ -1,12 +1,12 @@
 from enum import Enum
-from typing import Any, Dict, List, Optional
```

Comment:
You need to return this. In python version <3.9 there is no default support for `list`, `dict` and etc.

Reply:
Can't you just 
from __future__ import annotations ?

Reply:
Also it is not needed, cause it's a stub file and type checkers always treat those files with that import statement